packages/cloud-cognitive/src/components/CreateModal/_create-modal.scss

```diff
@@ -0,0 +1,72 @@
+//
```

Comment:
This file needs to use the import once approach. All of your sass should be wrapped in a mixin and you'll then include the mixin at the end of the file. Check out the `_example-component.scss` for reference.